src/NetMQ/SocketOptions.cs

```diff
@@ -12,6 +12,16 @@ namespace NetMQ
     /// </summary>
     public class SocketOptions
     {
+        /// <summary>
+        /// If set, the time stamp counter is not read directly through opcode injection,
+        /// rather <see cref="System.Diagnostics.Stopwatch.GetTimestamp"/> is used.
+        /// When false, the time stamp counter is read by allocating a few bytes on the heap with
+        /// read/write/execute privilege. OpCode is copied to this allocated memory and invoked to read
+        /// the time stamp counter, (which is a register available on most modern CPUs). While this is
+        /// an accurate way to read the time stamp counter, because it injects code onto the heap, this
+        /// can be detected as a malware technique by some anti-virus defenders.
+        /// </summary>
+        public static bool DoNotUseRDTSC;
```

Comment:
This doesn't really seem like a socket option, and we don't have any other static options here as far as I can tell.

Also the timing of setting this value is important, as the result of `Open` is cached in `Clock.s_rdtscSupported`.

Because of the timing issue, what about using something like an environment variable to control this. For example, if `NETQM_SUPPRESS_RDTSC` was present, it would disable any attempt to call `Opcode.Open()` and store a `false` in `s_rdtscSupported`. All that could happen in the static constructor of `Clock`.